packages/quarto-core/src/document.ts

```diff
@@ -100,6 +100,7 @@ export function isQuartoYaml(doc: Document) {
   return (
     doc.languageId === kYamlLanguageId &&
     (doc.uri.match(/_quarto(-.*?)?\.ya?ml$/) ||
+      doc.uri.match(/_brand\.ya?ml$/) ||
```

Comment:
In case anyone is curious about the lack of a Quarto version check here: if the pattern in `LanguageClientOptions` doesn't match, VS Code never actually handles `_brand.yml` to Quarto's LSP. A version check would be redundant.